test-engineering/contract-tests/client/README.md

```diff
@@ -27,14 +27,33 @@ The client is instructed on request and response check actions via scenarios,
 recorded in the `scenarios.yml` file. A scenario is defined by a name, a description 
 and steps.
 
-### Steps
+#### Name
 
-#### Kinto Service
+A test name should identify the use case under test . Names are written in snake case
+with double-underscores `__` for scenario and behavior delineation.
+
+Example:
+`remote_settings__refresh`
+
+#### Description
+
+A test description should outline, in greater detail, the purpose of a test, meaning
+the feature or common user interaction being verified.
+
+Example:
+_Test that Merino successfully returns refreshed output in the cases of
+suggestion content updates and additions_
```

Comment:
What are the leading and trailing `_` for?

Reply:
Italics

<img width="459" alt="image" src="https://user-images.githubusercontent.com/3422688/199612244-4e97f003-a6f5-4693-8303-233c8455305d.png">